notebooks/en/rag_with_hf_and_milvus.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p>Perhaps worth clarifying/a rule of thumb for what "large scale of data" is?</p><p> </p><p>"If you have a large amount of data, you can set up a more performant Milvus server on Docker or Kubernetes..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>